meetings/2024-11-20.md

```diff
@@ -0,0 +1,61 @@
+# Node.js  TypeScript team Meeting 2024-11-20
+
+## Links
+
+* **Recording**: <https://www.youtube.com/watch?v=cB_mOypkbIo>
+* **GitHub Issue**: <https://github.com/nodejs/typescript/issues/18>
+
+## Present
+
+* Typescript team: @nodejs/typescript
+* Marco Ippolito: @marco-ippolito
+* Andrew Branch: @andrewbranch
+* Rob Palmer: @robpalme
+* Jacob Smith @JakobJingleheimer
+* Daniel Rosenwasser @DanielRosenwasser
+* Augustin Mauroy @AugustinMauroy (observator)
+
+## Agenda
+
+## Announcements
+
+* Extracted from **typescript-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+    Jacob: The correct specifiers codemod has successfully migrated the Puppeteer monorepo, so I think it's ready to go (JakobJingleheimer/correct-ts-specifiers#9). I need to add a few test cases and polish docs, but I expect it to be ready to publish in the next few days.
+
+### nodejs/loaders
+
+* Roadmap for experimental TypeScript support [#217](https://github.com/nodejs/loaders/issues/217)
+
+### nodejs/typescript
+
+* Type Stripping in `node_modules/*/*.ts` [#14](https://github.com/nodejs/typescript/issues/14)
+    Marco:  No need to support it for the unflagging, it can wait.
+    Daniel: Right now don’t think it’s a good idea, but not definitive.
+    Jacob: use engines section of package.json to mitigate some the incompatibility issues
+    Marco: probably enable it behind a flag
+    Daniel: monorepo use symlinks so don’t need it
+    Andrew: monorepo already work, maybe does not work when publishing .ts files internally
+    Daniel: If users wants to use workspaces, that is already documented
+    Jacob: monorepo in fact works, tested in codemod
+    Marco: We believe monorepo use-cases should all work, we will re-evaluate after the feature becomes more stable.
```

Comment:
This does not format correctly in markdown (it collapses into a single continuous paragraph), but I don't know how to get it to do so.